### PR TITLE
Code First: backend API

### PR DESCRIPTION
**Pull request by srg-kostyrko**
API:

POST to /users
PUT to /users/:userId
GET /users?email=someuser@gmail.com

### CODE REVIEW

**Review by babblet — APPROVED** [review]
Looks good to me, tested and works as intended! Nice!

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
+import "dotenv/config";
+import express, { Request, Response } from "express";
+import cors from "cors";
+import { drizzle } from "drizzle-orm/libsql";
+import { usersTable } from "./db/schema.js";
+import { randomBytes } from "node:crypto";
+import { eq } from "drizzle-orm";
+import { LibsqlError } from "@libsql/client";
+
+const db = drizzle(process.env.DB_FILE_NAME!);
```
Comment (marckalil) [inline src/index.ts:10]: I think this should belong to your `db` folder to keep all database related setups, schemas, helpers, ... at the same place

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  EmailAlreadyInUse = "EmailAlreadyInUse",
+}
+
+const app = express();
+app.use(express.json());
+app.use(cors());
+
+// Create a new user
+app.post("/users/new", async (req: Request, res: Response) => {
+  if (!req.body || !isValidUsedData(req.body)) {
```
Comment (marckalil) [inline src/index.ts:26]: I think `!req.body` could be handled by `isValidUsedData`

**Review by marckalil — COMMENTED** [review]
I pointed out very little details.
Good job :)

**Pull request merged**